packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/PublicationState/tests/PublicationState.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { IntlProvider } from 'react-intl';
+
+import { PublicationState } from '..';
+
+const ComponentFixture = (props) => (
+  <ThemeProvider theme={lightTheme}>
+    <IntlProvider locale="en" messages={{}}>
+      <PublicationState {...props} />
+    </IntlProvider>
+  </ThemeProvider>
+);
+
+const setup = (props) => render(<ComponentFixture {...props} />);
+
+describe('DynamicTable | PublicationState', () => {
+  test('render draft state', () => {
+    const { container, getByText } = setup({ isPublished: false });
+
+    const statusNode = container.children[0];
```

Comment:
maybe worth writing a little note why you did it this way? took me a few moments to figure it out I would have initially thought "we should use `getByText`" for example which wouldn't work anyway.

Reply:
Done in caa30d8fe6c831de59e0bf5c1418174d32764727. I've also refactored it to use `.firstChild` as this is the recommended way: https://testing-library.com/docs/react-testing-library/api/#container-1